src/commands/new/glee.ts

```diff
@@ -1,55 +1,39 @@
-import {Flags} from '@oclif/core';
-import { promises as fPromises } from 'fs';
-import Command from '../../base';
-import { resolve, join } from 'path';
-import fs from 'fs-extra';
+import { Flags } from "@oclif/core";
+import Command from "../../base";
 
 export default class NewGlee extends Command {
-  static description = 'Creates a new Glee project';
```

Comment:
can you reset the formatting changes? it makes it hard to review.

Reply:
Actually this was my default setting.